package.json

```diff
@@ -90,6 +95,9 @@
     "@types/jasminewd2": "2.0.3",
     "@types/lodash": "^4.14.118",
     "@types/node": "^12.11.1",
+    "@typescript-eslint/eslint-plugin": "4.28.2",
+    "@typescript-eslint/parser": "4.28.2",
+    "eslint": "^7.26.0",
     "codelyzer": "^5.1.2",
```

Comment:
I think codelyzer is not needed anymore. As far as i know it was used by tslint.